test/unit/error.test.js

```diff
@@ -89,34 +110,6 @@ describe('MongoErrors', () => {
     });
   });
 
-  describe('#isRetryableEndTransactionError', function () {
```

Comment:
I'm not saying we should definitely test the new error helpers (isRetryableRead/Write), but what were your reasons for not including the unit tests for them?

Reply:
I can write some unit tests, no reason against them